R/SS_read.R

```diff
@@ -5,29 +5,28 @@
 #' Functionality comes from the `r4ss::SS_read*()` functions.
 #' This function simplifies the number of lines of code you need to write by
 #' using all of the read functions to read in the
-#' starter, control, data, and forecast files.
-#' The starter file is helpful because it provides names for the
-#' control and data files.
+#' starter, control, data, and forecast files and if requested, the
+#' weight-at-age file. The starter file is helpful because it provides names for
+#' the control and data files.
 #'
-#' @template dir
+#' @param dir A file path to the directory of interest or a raw github URL (see
+#' example). The default value is \code{dir = NULL}.
```

Comment:
@k-doering-NOAA I am wondering if we should take this opportunity to uncover the hidden argument of `dir = NULL` where it actually leads to `dir = getwd()`. Best practices suggests avoiding hidden arguments and just setting them in the function call.

Reply:
that is fine by me! I think that makes sense.